hoover/site/events.py

```diff
@@ -39,6 +39,17 @@ def on_doc(sender, request, collection, duration, success, **kw):
             success=success,
         )
 
+    @receiver(search_signals.batch)
+    def on_batch(sender, request, collection, duration, success, queries, **kw):
```

Comment:
Does this code even run? You're sending `collections` ([views.py line 158](https://github.com/hoover/search/pull/11/files#diff-57e9d1353205bd40aceee469eaa0184dR158)) but expecting `collection` here.